modules/Microsoft.ContainerInstance/containerGroups/deploy.bicep

```diff
@@ -49,6 +102,46 @@ var identity = identityType != 'None' ? {
   userAssignedIdentities: !empty(userAssignedIdentities) ? userAssignedIdentities : null
 } : null
 
+var dnsConfig = !empty(dnsNameServers) ? {
```

Comment:
Is there a reason we have entire variables for that even though we could do this directly in the properties block?

Reply:
see comment above.

Reply:
done